pages/features/online-payments.vue

```diff
@@ -0,0 +1,279 @@
+          heading: 'Event Registration with Fee Collection',
+          description:
+            'Organizers can effortlessly create customized registration forms, ensuring instant collection of fees and improved participant registration experience.',
+          imgUrl: 'features/online-payment/event-registration.png',
+        },
+        {
+          heading: 'Product Order and Payment',
+          description:
+            'Easily create order forms with integrated secure payment gateways, boosting e-commerce capabilities for processing online orders and payments.',
+          imgUrl: 'features/online-payment/product-order.jpg',
+        },
+        {
+          heading: 'Membership Subscription Renewal',
+          description:
+            "Craft membership renewal forms with integrated payment options. This ensures an efficient process for members to renew their subscriptions online.",
+          imgUrl: 'features/online-payment/membership.jpg',
+        },
+        {
+          heading: 'Donation Collection for Nonprofits',
+          description:
+            'Create nonprofit donation forms, allowing supporters to contribute online. This streamlines the donation process and provides a reliable payment platform to the donors.',
+          imgUrl:
+            'features/online-payment/donation.jpg',
+        },
+        {
+          heading: 'Service Request for Professionals',
+          description:
+            'Build service request forms that include payment options. Clients can submit their requests and make payments through the online form, simplifying the transaction process for both parties.',
+          imgUrl:
+            'features/online-payment/service-request-for-professionals.jpg',
+        },
+      ],
+      CallToActionContent: {
+        heading: "Create your first online payment form!",
+      },
+      threeStepContent: {
+        heading: "How to activate online payments in 3 simple steps:",
+        steps: [
+          {
+            title: "Sign up",
+            description: "Sign up for a free account on formester and build an online form. You can also select any template from our pre-designed templates library."
+          },
+          {
+            title: "Payment integration selection",
+            description: "Select a payment integration option in the form builder and drag & drop it to your form."
+          },
+          {
+            title: "Form sharing",
+            description: "Share the form and start collecting payments along with form submissions."
+          },
+        ]
+      },
+      templateContent: {
+        heading: "Free Online Payment Form Templates",
+        slug:"order-forms"
+      }
+    }
+  },
+  async asyncData() {
+    let randomTestimonials = await allTestimonials
+    const randIndex = Math.floor(Math.random() * (randomTestimonials.length - 2))
+    randomTestimonials = randomTestimonials.slice(randIndex,  randIndex + 2);
+    return {randomTestimonials}
+  },
```

Comment:
this can be replaced with the utils as well

Reply:
got it